### ISSUE TITLE
Lazy loading

### ISSUE THREAD

**Issue by muzrix** [soflyy/oxygen-bugs-and-features#163]
**Describe the feature you'd like to see included in Oxygen.**
Have an option to set certain elements (section/div/column or image/video) to load only on viewport.

**What are the use cases for this feature?**
- Will Increase performance
- Will save data for visitors with limited data plan especially on mobile.

**Examples of this feature or functionality.**
It just make sense not to load what user don't see.

**Comment by totemdomains** [comment]
yes, please...considering converting a client's site over from Divi to Oxygen and this would be most welcome! (artist's site with heavy galleries, etc.)

**Comment by TnowL** [comment]
I recommend Lazy Load by WP Rocket plugin for this.

**Comment by wpdevlab** [comment]
+1

**Comment by PaRoxUs** [comment]
+1, it would be great to have this enabled with Oxygen together with integration for optimized images, that would be an option to resize images in the backend, so they fit perfect for different screen sizes including retina displays

**Comment by roireshef** [comment]
+1, I think this is a basic functionality...

**Comment by Spellhammer** [comment]
Images now have a lazy load option, but we should probably add it to Galleries and Video elements as well.

**Comment by Bebuk85** [comment]
This is the best way to lazy load elements:

[https://web.dev/content-visibility/
](https://web.dev/content-visibility/
)
Flyingpress already did this
![image](https://user-images.githubusercontent.com/72046610/214950977-e6888ef5-fe72-44ac-8bf4-9fef2231a830.png)

**Comment by QuirkyRobots** [comment]
Content Visibility is good, but it can make page pages jump when scrolling.
For images, videos and iFrames, `loading="lazy"` is best. For everything else, `<template>Content Here</template>` activated on by JS when coming in to view is best. This is what Vue JS does.
